constants.py

```diff
@@ -65,6 +65,10 @@
 
 # Defaults to 0.85
 CHARGER_PLUS_CAR_ROUNDTRIP_EFFICIENCY: float = 0.85
+# Defaults to min current setting of 6A * 240V = 1440W
+CHARGER_MAX_CHARGE_POWER: int = 1440
```

Comment:
Why so low? I expected 11A to be a more useful default.

Also, I heard some cars "fall asleep" below 6A, so this setting would be taking it right to the edge of what's possible (for some cars).

Reply:
This is just to be on the safe side. If a user makes a typo in the secrets file we keep the power low.